HTTPlayground/base.py

```diff
@@ -0,0 +1,46 @@
+from http.cookies import SimpleCookie
+from pathlib import Path
+from HTTPlayground.settings import BASE_DIR, TEMPLATE_DIR, STATIC_URL
+
+
+def send_headers(request, response_code: int = 200, content_type: str = 'text/plain', **kwargs):
+    request.send_response(response_code)
+    request.send_header('Content-type', content_type)
+    for k, v in kwargs.items():
+        request.send_header(k, v)
+    request.end_headers()
+
+
+class CookieHandler:
```

Comment:
from what I see, using dict param is an overhead here:
1. you use both methods in the class for a single key-value pair anyways
1. named params will be more readable in this case and you won't need to trickily unpack it (lines 19, 29 so far), also, there won't be need for f-strings, because the key will be a string by method definition.